recipes-core/aziot-edged/files/iotedge.conf

```diff
@@ -0,0 +1 @@
+d  /var/run/iotedge 0755 iotedge iotedge -
```

Comment:
What is the purpose of this .conf file?

Reply:
I was seeing errors like this during the aziot-edged service initialization:
    aziot-edged[6879]: 2023-01-19T21:37:59Z [ERR!] - Could not start listener for module edgeAgent, error Failed to listen on workload socket: Permission denied (os error 13)

This occurs because aziot-edged.mgmt.socket runs as root during boot and creates the /var/run/iotedge directory as 755, root:root.

I've resolved the issue by having systemd-tmpfiles.setup.service (via /etc/tmpfiles.d) create it first as 755, iotedge:iotedge.